spec/features/wiki/crud_spec.rb

```diff
@@ -13,9 +13,9 @@
     click_button "Log in"
 
     # localhost/wikis/new  wikis#index
-    expect(page).to have_content("Hello test@email.com")
+    expect(page).to have_content("Hey test@email.com")
     expect(current_path).to eq(wikis_path)
-    click_link "New wiki"
+    click_link "Public wiki" || "Private wiki"
```

Comment:
The public and private wiki features are different enough that I'd consider having separate scenarios for them, or even separate `*_spec.rb` files. First spec structure that springs to mind:

```ruby
# spec/features/create_wiki_spec.rb
feature "create wiki" do
  xscenario "public wiki" do
    
  end

  context "private wiki"
    xscenario "cannot be created by standard user" do
    end
    
    xscenario "can be created by premium user" do
    end
  end

end
```